src/pkgcore/sync/rsync.py

```diff
@@ -143,7 +143,7 @@ def _sync(self, verbosity, output_fd):
 
         # zip limits to the shortest iterable
         ret = None
-        for count, ip in zip(range(self.retries), self._get_ips()):
+        for _, ip in zip(range(self.retries), self._get_ips()):
```

Comment:
Will `itertools.islice` work here?

Reply:
That it would, thanks.  I'll update shortly.